src/greatwall/resources/knowledge/mnemonic/GUI_qt.py

```diff
@@ -559,8 +559,10 @@ def config_tab(self):
         self.reset_button.clicked.connect(self.reset_selection)
         valid_message = "Selected phrase\nso far is compatible"
         self.sel_valid_phrase_label.setText(valid_message)
+        
```

Comment:
Keep this file as it is, this file is not related to our GUI and we will remove it in the future.